### PR TITLE
PR for "Rework Building a Connector images"

### PR DESCRIPTION
**Pull request by Beccamak**
@svrnm I trust your day is going well, mentor. This is the PR for this issue #5358 
I also made some changes to the SVG file based on your suggestion. Please look through it when you can. 
Thank you so much for always taking the time to review things. 
I'm looking forward to your feedback.

### CODE REVIEW

File: content/en/docs/collector/deployment/otel-collector-before-connector.png
Comment (svrnm) [inline content/en/docs/collector/deployment/otel-collector-before-connector.png:1]: You can delete the old files

Comment (Beccamak) [reply content/en/docs/collector/deployment/otel-collector-before-connector.png:1]: @svrnm Thank you so much. I'll delete the files, remove the "Same Connector" text, and make the line go between the pipelines to see if it's better.

Comment (Beccamak) [reply content/en/docs/collector/deployment/otel-collector-before-connector.png:1]: @svrnm I was able to view the screenshot you sent to me yesterday, but I'm unable to view it now. 
Please take a look at the SVG files and see if you're okay with any. Thank you.
New Architecture (1)
New Architecture (2)

Comment (svrnm) [reply content/en/docs/collector/deployment/otel-collector-before-connector.png:1]: I like the second option

File: content/en/docs/collector/img/otel-collector-after-connector.svg.svg
Comment (svrnm) [inline content/en/docs/collector/img/otel-collector-after-connector.svg.svg:1]: let's use this one

Comment (Beccamak) [reply content/en/docs/collector/img/otel-collector-after-connector.svg.svg:1]: @svrnm Roger that.

Comment (Beccamak) [reply content/en/docs/collector/img/otel-collector-after-connector.svg.svg:1]: @svrnm I have made the changes. Please look through it. Thank you.

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
@open-telemetry/collector-approvers please take a look!

**Comment by svrnm** [comment]
@Beccamak while we wait for @open-telemetry/collector-approvers to take a look, you can treat this PR as "done" for your outreachy application.

**Comment by Beccamak** [comment]
@svrnm Thank you so much. I'll update my contribution.

**Review by djaglowski — APPROVED** [review]
Nice improvement

**Comment by svrnm** [comment]
@Beccamak I prepare your PR for being merge, I just did a mistake myself, will fix it and have it ready then, nothing for you to worry about

**Comment by Beccamak** [comment]
@svrnm Wow!!! Thank you so much 🙏 I'm so sorry about the mistake.

**Pull request merged**